### DISCUSSION TITLE
main_callable always uses the inherited __init__

### DISCUSSION THREAD

**Discussion by ghost** [IndiePython/nodezator#93] [Concerns/constructive criticism]
I can't seem to get Nodezator to use my overriden __init__ func as it always defaults to the superclasses __init__. Can be worked around, but many times it disallows me to use the Callable mode further down the line.

**Comment by KennedyRichard** [comment]
Since you already closed this discussion, I think it is reasonable to assume you already dealt with the problem, so I won't comment further. If there's still anything you need clarified though, just let me know.
